--- origsrc/rubysdl-2.0.1b/extconf.rb	2008-07-20 02:46:30.000000000 -0500
+++ src/rubysdl-2.0.1b/extconf.rb	2008-09-04 21:01:02.929783500 -0500
@@ -74,7 +74,7 @@
   
   $CFLAGS+= " -D ENABLE_OPENGL "
 
-  if /linux/ =~ CONFIG["arch"] then
+  if /linux|cygwin/ =~ CONFIG["arch"] then
     have_library("GL","glVertex3d")
   elsif /mingw32/ =~ CONFIG["arch"] then
     have_library("opengl32","glVertex3d")
